ktlint/build.gradle

```diff
@@ -44,3 +44,20 @@ dependencies {
   testImplementation deps.junit
   testImplementation deps.assertj
 }
+
+tasks.create("shadowJarExecutable") {
+  description = "Creates self-executable file, that runs generated shadow jar"
+  group = "Distribution"
+
+  inputs.files tasks.named("shadowJar")
+  outputs.file "$buildDir/run/ktlint"
+
+  doLast {
```

Comment:
Would be nice to add a link to either the maven plugin or the blog post that inspired it, just so it's clear what we're doing here.

Reply:
Added comments above task defintion.